### PR TITLE
Issue template

### PR DESCRIPTION
**Pull request by ferrine**
This PR simplifies opening issues at this repo

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: PyMC Discourse
+    url: https://discourse.pymc.io/
+    about: Ask usage questions about PyMC/PyMC3
+  - name: Example notebook error report
+    url: https://github.com/pymc-devs/pymc-examples/issues
+    about: Please report errors or desired extensions to the tutorials and examples here.
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
```

Do we need this? We can build example notebooks in the docs build and fail it automatically on error: https://github.com/pymc-devs/pymc/pull/6324

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# This issue template was adapted from the NumPy project
+# under the BSD 3-Clause "New" or "Revised" License.
+# Copyright (c) 2005-2022, NumPy Developers.
+# All rights reserved.
+
+
+name: Documentation
+description: Report an issue related to the PyMC documentation.
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/documentation.yml:8]:
```suggestion

name: Documentation
description: Report an issue related to the Pytensor documentation.
```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+body:
+- type: textarea
+  attributes:
+    label: "Issue with current documentation:"
+    description: >
+      Please make sure to leave a reference to the document/code you're
+      referring to. You can also check the development version of the
+      documentation and see if this issue has already been addressed at
+      https://www.pymc.io/projects/docs/en/latest/api.html.
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/documentation.yml:20]:
```suggestion
      https://pytensor.readthedocs.io/en/latest/.
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# This issue template was adapted from the NumPy project
+# under the BSD 3-Clause "New" or "Revised" License.
+# Copyright (c) 2005-2022, NumPy Developers.
+# All rights reserved.
+
+name: Bug report
+description: Report a bug. For security vulnerabilities see Report a security vulnerability in the templates.
+title: "BUG: <Please write a comprehensive title after the 'BUG: ' prefix>"
+labels: [defects]
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/bug-report.yml:9]:
```suggestion
labels: [bug]
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- type: markdown
+  attributes:
+    value: >
+      If you're looking to request a new feature or change in functionality,
+      including adding or changing the meaning of arguments to an existing
+      function, please post your idea first on please create a discussion thread at
+      [Discussions](https://github.com/pymc-devs/pytensor/discussions)
+      to validate it and bring attentino to it. After validation,
+      you can open this issue for a more technical developer discussion.
+      [Contributor Guide](https://github.com/pymc-devs/pytensor/blob/main/CONTRIBUTING.md)
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/feature-request.yml:22]:
```suggestion
      Check the [Contributor Guide](https://github.com/pymc-devs/pytensor/blob/main/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    [Discussions](https://github.com/pymc-devs/pytensor/discussions)
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: "Before"
+  descrition: >
+    Please fill the code snippet how did you workaround your problem or frequent use.
+    Leave empty if you see it impossible
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/feature-request.yml:39]:
```suggestion
    Please fill the code snippet: How did you workaround your problem or frequent use?
    Leave empty if you found no workaround.
```

File: .github/ISSUE_TEMPLATE/installation.yml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+# This issue template was adapted from the NumPy project
+# under the BSD 3-Clause "New" or "Revised" License.
+# Copyright (c) 2005-2022, NumPy Developers.
+# All rights reserved.
+
+
+name: Post-install/importing issue
+description: Report an issue if you have trouble importing or using PyMC after installation.
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/installation.yml:8]:
```suggestion
description: Report an issue if you have trouble importing or using Pytensor after installation.
```

**Comment by ferrine** [comment]
Thank you @michaelosthege I did miss a lot copy pasting templates from pymc

**Review by michaelosthege — APPROVED** [review]
Let's see how it turns out

**Pull request merged**